src/model/consent/consent.ts

```diff
@@ -0,0 +1,105 @@
+/* istanbul ignore file */
+
+/*
+ * This flag is to ignore BDD testing for model
+ * which will be addressed in a future ticket
```

Comment:
Please open ticket and reference explicitly here

Reply:
I have added #354 ticket with basic goal on the PISP board. Please let me know if that works. Do you want me to just mention the ticket number here?